tests/unit/addon-tree-test-node.js

```diff
@@ -84,15 +105,19 @@ describe('Addon', function() {
             contentFor = addon.contentFor('ember-index-3');
 
             expect(contentFor).to.be.equal(addon._startMarkerPrefix + '-3' + '<!-- content from file3 -->' + addon._endMarkerPrefix + '-3');
+
+            contentFor = addon.contentFor('ember-index-4');
+
+            expect(contentFor).to.be.equal(addon._startMarkerPrefix + '-3' + '<!-- content from string -->' + addon._endMarkerPrefix + '-3');
```

Comment:
You have a typo( `-3` instead of `-4`). change the line to:
```javascript
contentFor = expect(contentFor).to.be.equal(addon._startMarkerPrefix + '-4' + '<!-- content from string -->' + addon._endMarkerPrefix + '-4');
```